prisma/schema.prisma

```diff
@@ -99,31 +99,32 @@ model CollectionAuthor {
 }
 
 model Label {
-  id          Int       @id @default(autoincrement())
-  externalId  String    @default(uuid()) @db.VarChar(255)
-  name        String    @unique @db.VarChar(255)
-  createdAt   DateTime  @default(now()) @db.DateTime(0)
-  createdBy   String    @db.VarChar(255)
-  updatedAt   DateTime  @updatedAt
-  updatedBy   String?   @db.VarChar(255)
+  id          Int               @id @default(autoincrement())
+  externalId  String            @default(uuid()) @db.VarChar(255)
+  name        String            @unique @db.VarChar(255)
+  createdAt   DateTime          @default(now()) @db.DateTime(0)
+  createdBy   String            @db.VarChar(255)
+  updatedAt   DateTime          @updatedAt
+  updatedBy   String?           @db.VarChar(255)
+  collections CollectionLabel[]
+
 
   @@unique([externalId])
   @@index([id])
   @@index([name])
-
 }
 
 model CollectionLabel {
-  id                    Int @id @default(autoincrement())
```

Comment:
I think we should keep this field and use it instead of the composite unique ID proposed in this PR. We tried this approach earlier, I think with `CollectionPartnership` entity and Prisma did not take it well at the time (I wish I could remember the details! Types I think were the main problem) - things might have changed so you could still go ahead and try it if you like, but this would be the first place to look if things don't work as expected.

Reply:
@nina-py do you remember in what scenarios this might have caused problems before? my mind is unfortunately blank on this 😄.

my vote is to try with the composite id. we should get feedback pretty early when we test the associated resolvers. it would be nice to not have an extra `id` field - particularly based on the intended usage of this table, which will likely have a lot of deletes and inserts, but zero updates.

Reply:
my vote doesn't have to be the winner here though! would love others to chime in as well.